ts/Core/Pointer.ts

```diff
@@ -1275,6 +1268,11 @@ class Pointer {
         ) {
             this.reset();
         }
+
+        // #15160
+        if (H.hasTouch) {
+            this.chart.series.forEach((series: Series): void => series.setState());
```

Comment:
Did you consider doing this in the `Pointer.reset` function? In general, the job of that function is to "Reset the tracking by hiding the tooltip, the hover series state and the hover point". It is responsible of resetting the state of the `hoverSeries`:

https://github.com/highcharts/highcharts/blob/41e481abd12743502dcc1600ed70d0814e0ace5b/ts/Core/Pointer.ts#L1823-L1825

So maybe that's where we should add the code to reset inactive states?

... And why does this apply only to touch devices?

Reply:
Because the `mouseout` handler on the legend item resets the state for non-touch devices. Doing it in `reset` seems to work a bit better since it gets called when clicking outside the plot also.

Reply:
`mouseout` normally fires when clicking outside elements after clicking them on touch devices, not sure why it doesnt happen here. The container `mouseleave` also doesnt get fired if you only click a legend item and then outside the chart, which means `reset` doesnt get called in that case.

Reply:
Legend `mouseout` currently sets inactive items state to '', and also sets item state to ''. It may be that it would be smarter to just call `pointer.reset` from legend `mouseout`. I think that's essentially what we are doing anyway.

Reply:
.. and then we wouldn't have to double-check for 'H.hasTouch'.

Reply:
But the problem is the `mouseout` doesnt fire on touch devices because of this: https://github.com/highcharts/highcharts/blob/a55cae7ddaa73936a32fe430b83eff1e5dc4341e/ts/Core/Renderer/SVG/SVGElement.ts#L2128, #9682.

Reply:
Could we perhaps get rid of the 300ms click delay fix since I believe this is no longer an issue in modern browsers?

Reply:
> But the problem is the mouseout doesnt fire on touch devices

Won't touching anywhere outside the plot area trigger `Pointer.reset` on touch devices? So if we centralize this inactive `setState` to `Pointer.reset`, then it will be handled there regardless of legend `mouseout`?

> Could we perhaps get rid of the 300ms click delay fix

Maybe... Where is that?

Reply:
No, touching a legend item and then outside the chart container wont fire the container `mouseleave` so `reset` wont be called. This is because of the `preventDefault` I linked above, its part of the click delay fix.

Reply:
Okay... Then I agree that's where we should look for a solution. If we can avoid the `preventDefault` and still avoid the linked issue, it's the most consistent fix I think.